R/moduleInstall.R

```diff
@@ -1,12 +1,338 @@
+postInstallFixes <- function(folderToFix) {
+  if(length(ls(all.names=TRUE,pattern="\\..postProcessLibraryModule")) > 0 ) #We seem to be running in JASP
+  {
+    #print("we are *in* jasp, so we use .postProcessLibraryModule!")
+    .postProcessLibraryModule(folderToFix)
+  }
+  else 
```

Comment:
Is the `else` for use outside of JASP? Any reason in particular we use system variables rather than an optional argument to `postInstallFixes`?

Reply:
Yeah when you are building JASP ^^

Reply:
Makes sense! ^.^